data/styles/main.css

```diff
@@ -82,30 +82,29 @@ nav .title {
 }
 
 #search-box {
+    background-clip: padding-box;
+    background-color: #fff;
+    border: 1px solid rgba(0,0,0,0.05);
+    border-radius: 3px;
     display: none;
-    width: auto;
 }
 
 #search-field {
-    background-clip: padding-box;
-    border: 1px solid rgba(0, 0, 0, 0.05);
-    border-radius: 3px;
+    background-color: transparent;
+    border: none;
     font-size: 14px;
-    padding: 6px;
-    transition: all 200ms ease-in-out;
```

Comment:
There wasn't anything that got animated. If we add some kind of animation in, then we can add bac the transition line ;)